earth_enterprise/src/scons/khEnvironment.py

```diff
@@ -236,149 +236,6 @@ def EmitLongVersionStrfunc(target, backupFile, label):
   return 'EmitLongVersion(%s, %s, %s)' % (target, backupFile, label)
   
 
```

Comment:
Moved all of the version-related code to its own file, and export only the functions used by khEnvironment (GetVersion, GetLongVersion).